tests/string_test.py

```diff
@@ -505,3 +505,17 @@ def test_string_registration(self):
         ak.clear()
         with self.assertRaises(RuntimeError, msg="keep was unregistered and should be cleared"):
             str(keep) # should cause RuntimeError
+
+    def test_string_is_registered(self):
```

Comment:
I think this should be included in the registration tests instead of here but I am open to a counter argument.

Reply:
@mhmerrill ah, good point, I agree.

Reply:
@mhmerrill @hokiegeek2 , if @pierce314159 moves this new test `test_string_is_registered` should the existing `test_string_registration` also go with it?

Reply:
yeah my reasoning for putting `test_string_is_registered` in `string_test.py` was that the `test_string_registration` method was already there. And that they are both methods of the `Strings` class. It makes just as much sense to me to put them in the registration tests. Should I move both or just `test_string_is_registered`?

Reply:
yes, move them